### PR TITLE
Issue #10985

### PR DESCRIPTION
**Pull request by kritkaran94**
@asmeurer Sorry for the addition of pyc files. Please review this PR.

### CODE REVIEW

**Comment by debugger22** [comment]
@kritkaran94 You should use some meaningful text in PRs/commits.

**Comment by kritkaran94** [comment]
@debugger22 Sorry for that. I will keep that in mind

File: sympy/printing/pretty/pretty.py [hunk 1]
```diff
@@ -811,11 +811,11 @@ def _print_BasisDependent(self, expr):
                 #if the coef of the basis vector is 1
                 #we skip the 1
                 if v == 1:
-                    o1.append(u("") +
+                    o1.append(u'' +
                               k._pretty_form)
                 #Same for -1
                 elif v == -1:
-                    o1.append(u("(-1) ") +
+                    o1.append(u('-1') +
```
Comment (asmeurer) [inline sympy/printing/pretty/pretty.py]: This is a different string.

File: sympy/printing/pretty/pretty.py [hunk 2]
```diff
@@ -828,7 +828,7 @@ def _print_BasisDependent(self, expr):
                 vectstrs.append(k._pretty_form)
 
         #outstr = u("").join(o1)
-        if o1[0].startswith(u(" + ")):
+        if o1[0].startswith(u'' + u''):
```
Comment (asmeurer) [inline sympy/printing/pretty/pretty.py]: This is a different string.

**Comment by asmeurer** [comment]
There's a lot of files that it was removed from, but no other changes were made. Did you verify that it wasn't actually used in any of those files? A good way to check would be to run `pyflakes` on the files and see if it reports any undefined `u` variables (or run it before removing the import and see if it reports an unused import).

**Comment by kritkaran94** [comment]
@asmeurer I didn't do that. I will proceed as you have just stated.

**Comment by gxyd** [comment]
Closing. Since the corresponding issue has been fixed by https://github.com/sympy/sympy/pull/11338.

**Pull request closed without merging**